dns_connector_dnspod/dnspod.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2015 Elico Corp (<http://www.elico-corp.com>)
+#    Authors: Liu Lixia, Augustin Cisterne-Kaas
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields, api
+from .backend import dnspod
+from .unit.export_synchronizer \
+    import DNSExporter
+from .unit.backend_adapter \
+    import DNSPodAdapter
+from openerp.addons.connector.unit.mapper import (
+    mapping, ExportMapper)
+import httplib
+import urllib
+import json
+
+
+class DNSPodBackend(models.Model):
+    _inherit = 'dns.backend'
+
+    def select_version(self):
+        """return version selection"""
+        res = []
+        res.append(('dnspod', 'dnspod'))
+        return res
+
+    version = fields.Selection(select_version, string='Service Provider',
```

Comment:
Not needed to redefine the field if the base fields correctly put `''` around `select_version`